app/src/test/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentLogicTest.kt

```diff
@@ -99,7 +101,7 @@ class ScheduleAppointmentLogicTest {
 
     verify(repository).schedule(
         patientUuid = eq(patientUuid),
-        appointmentUuid = any(),
+        appointmentUuid = eq(appointmentUuid),
```

Comment:
we can probably remove the `eq` and use the variables directly?

Reply:
We could, I'll do this.

Reply:
Done.